src/app/auth/PrivateRoute/index.tsx

```diff
@@ -1,22 +1,21 @@
 import React from "react";
 import { Route, Redirect } from "react-router-dom";
 
-const PrivateRoute = ({ component: Component, ...rest }) => (
-  <Route
-    {...rest}
-    render={props =>
-      localStorage.getItem("currentUser") ? (
-        <Component {...props} />
-      ) : (
-        <Redirect
-          to={{
-            pathname: "/login",
-            state: { from: props.location }
-          }}
-        />
-      )
-    }
-  />
-);
+const PrivateRoute = ({ component: Component, ...rest }) => {
+  return (
+    <Route
+      {...rest}
+      render={props =>
+        localStorage.getItem("currentUser") ? (
```

Comment:
Why you use `localStorage`, instead of redux? This is anti-pattern of React-Redux, because everything that you can move to redux, must be moved to redux. Otherwise it could break immutable logic of application.

Reply:
The current user state needs to be persisted, which is why it's used here. The exact implementation of this may change as we implement the real oauth flows, but this is fine for now.

Reply:
And? In my first PR, I fixed this, and everything work well through Redux, why not to keep this?
https://github.com/bond95/mig-ui/blob/5971ea8f81c094e7b6259e7d3588d77a56ba545b/src/components/PrivateRoute/index.tsx#L4-L16

Reply:
authentication did not persist through a page refresh with your implementation

Reply:
This is strange because this work, could you, please, in detail describe why it's not working for you?